src/controllers/PointModeController.js

```diff
@@ -47,13 +47,14 @@ class PointModeController extends AnnotationModeController {
 
         // Determine if a point annotation dialog is already open and close the
         // current open dialog
-        const hasPendingThreads = this.destroyPendingThreads();
-        if (hasPendingThreads) {
+        this.hasPendingThreads = this.destroyPendingThreads();
```

Comment:
Based on these method names I would always expect this.hasPendingThreads to be false after calling destroyPendingThreads, is that not the case?

Reply:
destroyPendingThreads returns whether or not any pending threads were existed on the file. 
Changing the variable to hadPendingThreads to be clearer